lib/bot_framework/dialogs/action_set.rb

```diff
@@ -0,0 +1,59 @@
+module BotFrameork
+  module Dialogs
+    class ActionSet
+      attr_accessor :actions
+
+      def initialize()
+        @actions = {}
+      end
+
+      def add_dialog_trigger(actions,dialog_id)
+        if trigger
```

Comment:
Favor modifier `if` usage when having a single-line body. Another good alternative is the usage of control flow `&&`/`||`.